app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy, foreign_key: 'author_id'
+  has_many :comments, dependent: :destroy, foreign_key: 'author_id'
+  has_many :likes, dependent: :destroy, foreign_key: 'author_id'
+
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def three_recent_posts
+    posts.order('created_at DESC').limit(3)
+  end
```

Comment:
- [ ] It is required that any custom methods in your model should have a corresponding unit test. Please include a test case for this method `three_recent_posts`.

Reply:
done